dll/win32/browseui/addressband.cpp

```diff
@@ -415,8 +415,17 @@ LRESULT CAddressBand::OnTipText(UINT idControl, NMHDR *notifyHeader, BOOL &bHand
 {
     if (notifyHeader->hwndFrom == fGoButton)
     {
-        // TODO
-        // Go to "destination path"
+        WCHAR szText[MAX_PATH];
+        WCHAR szFormat[MAX_PATH];
+        LPNMTBGETINFOTIP pGIT = (LPNMTBGETINFOTIP)notifyHeader;
+
+        if (::GetWindowTextW(fEditControl, szText, MAX_PATH))
+        {
+            LoadStringW(_AtlBaseModule.GetResourceInstance(), IDS_GOBUTTONTIPTEMPLATE, szFormat, MAX_PATH);
+            wnsprintf(pGIT->pszText, MAX_PATH, szFormat, szText);
```

Comment:
Please use `pGIT->cchTextMax`, because it is not `MAX_PATH`. (It will almost always be `INFOTIPSIZE`